README.md

```diff
@@ -0,0 +1,67 @@
+# Bambora
+
+The official Bambora Ruby library is not thread-safe. This means you will run into errors when using it with Sidekiq or Puma. This gem is a thread-safe client for the Bambora and Beanstream APIs.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'bambora-client'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install bambora-client
+
+## Usage
+
+You can initilize the client by passing a hash to it, or by passing a block.
+
+```ruby
+client = Bambora::Client.new(
+  api_key: ENV.fetch('BAMBORA_API_KEY'),
+  merchant_id: ENV.fetch('BAMBORA_MERCHANT_ID'),
+)
+```
+
+```ruby
+client = Bambora::Client.new do |c|
+  c.api_key = ENV.fetch('BAMBORA_API_KEY')
+  c.merchant_id = ENV.fetch('BAMBORA_MERCHANT_ID')
+end
+```
+
+### Sub-Merchant ID
+
+You can also initilze a client with a sub-merchant-id:
+
+```ruby
+client = Bambora::Client.new(
+  api_key: ENV.fetch('BAMBORA_API_KEY'),
+  merchant_id: ENV.fetch('BAMBORA_MERCHANT_ID'),
+  sub_merchant_id: 'submerchantid',
+)
+```
+
+## Development
+
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```

Comment:
Is it possible to line-break text to a certain amount of characters, so that it is easily readable in text editors as well as in GitHub Markdown?